### ISSUE TITLE
No Message-ID generated during SMTP transaction

### ISSUE THREAD

**Issue by ro78** [atech/postal#153]
Hello,

By default on SMTP servers, if a Message-ID is not provided by client, server is generating it and add it to headers.

With Postal, no Message-ID is added.

Example on email received on Gmail:
Message-Id: <5919a517.b482df0a.41610.750cSMTPIN_ADDED_MISSING@mx.google.com>

Romain

**Comment by adamcooke** [comment]
You are right, Postal doesn't generate and add message IDs. I've converted this into an issue to add server side message ID generation when one is missing.

**Comment by ro78** [comment]
@adamcooke Any update ?

**Comment by adamcooke** [comment]
No

**Comment by xjunin** [comment]
@adamcooke Any Update?

**Comment by adamcooke** [comment]
No

**Comment by xjunin** [comment]
@adamcooke the outgoing_message_prototype in folder app/models work only for message sending by api and web postal?
Have any files that interact directaly for smtp protocol?

thanks.

**Comment by pdifeo** [comment]
Hi, Any Update? There are some old applications written with .net asp and I have a big problem with receiver mail server. SpamAssassin and RSpamD assign a 2.5 score when Message-ID is missing.
